search-app/src/cmr/search/services/parameters/conversion.clj

```diff
@@ -197,6 +197,16 @@
    :concept-id :string
    :keyword :keyword})
 
+;; TODO: Generic work: See if we can wrap this in a doseq for all generic types or get from 
+;; a config file.
+(defmethod common-params/param-mappings :dataqualitysummary
+  [_]
+  {:name :string
+   :id :string
+   :provider :string
+   :native-id :string
+   :concept-id :string})
```

Comment:
Do we allow search for DataQualitySummary by Name only or by these other parameters also?  I don't see any document update in the search api.md.  Maybe it's time to write up something for the generic document.

Reply:
Yes, I agree, we will need to add documentation on how to add more concepts and how to search.